### PR TITLE
Feature/blog

### PR DESCRIPTION
**Pull request by anamariasosam**
@alexrqs please check 
- [ ] /blog
- [ ] /blog/post/1

TODO
- [ ] Add the name of the post in the route (/blog/post/nameOfThePost)

### CODE REVIEW

File: source/components/styles/PostItem.scss [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    &__description {
+      font-size: 0.85em;
+    }
+
+    &__title {
+      font-size: 1.4em;
+    }
+  }
+
+  @media (max-width:991px) {
```
Comment (alexrqs) [inline source/components/styles/PostItem.scss]: there's a respond to mixin onside source/style/mixin.scss import that at the top of the page :)

File: source/containers/RecentPosts.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      <section className="container">
+        <header>
+          <h3>Posts mas recientes</h3>
+        </header>
+
+        <div>
+          {/* TODO: Remove index and replace with product id after the real API */}
+          {this.state.posts.map((post, i) => (
+            <div key={i}>
+              <PostItem {...post} />
```
Comment (alexrqs) [inline source/containers/RecentPosts.jsx]: you can just add the key to the PostItem :)

Comment (anamariasosam) [inline source/containers/RecentPosts.jsx]: 👍

File: source/components/styles/PostItem.scss (hunk 1 above)
Comment (anamariasosam) [inline source/components/styles/PostItem.scss]: done

**Pull request merged**